### PR TITLE
TRLST-169 branding changes

### PR DESCRIPTION
**Pull request by jwgulley**
These are changes for making updates to the branding as described in TRLST-169.  Tipu has kindly reviewed the result and declared himself happy with it..  This PR is for review only at this stage, but with a view to it being approved and applied when the branding update is required.

Known Issues

These changes do not include:
- updates to any documents - e.g. letter of authority templates or email templates;
- updates to the database - see the ticket comment for details, also the fixture changes may be relevant.

### CODE REVIEW

**Review by markhigham — APPROVED** [review]
LGTM

**Review by davecharles — COMMENTED** [review]
Looks fine but could we use a combination of custom template tags (for the HTML) and calls to `get_system_parameters` (e.g. `Client().get_system_parameters('APP_BRAND')`? This means we can affect the change more easily at the correct time with a quick change in the apis `system/parameters.json`.

**Review by rossmiller — CHANGES_REQUESTED** [review]
Could we make the name of the product and its initialism a settings variable tied to an env var? That way we can change it any time.

**Comment by jwgulley** [comment]
I think this needs to be discussed.  The change will happen over the Christmas break, so my call was to keep it as simple as possible to avoid disrupting people's seasonal break.  Also, I think org name changes are actually likely to be rather rare.  Finally, the testing / reviewing work here is probably as substantial than the actual development.

It is worth noting that there are some file names referring to TRID - some might argue that a second set of changes will be needed in order to change those filenames and it would be better to do any automation at that point - i.e not during the holiday season.

**Comment by davecharles** [comment]
The intention is to minimise the change at that time (xmas) by parameterising it now, but using values for current branding (TRID etc). Then at EOY it's a simple var change that's low ceremony/low risk. My experience is that branding changes regularly enough (don't get me started 😆 ) to make this worth the effort now tbh.

**Comment by jwgulley** [comment]
Actually. there's more to say here.  If you are going to go down the route suggested, there is also a case for making a string table containing every string in the system.  The reason for doing this would be to support localisation - i.e. so that the web-site can implemented in different languages.

**Comment by rossmiller** [comment]
Sorry, I hadn't read @davecharles  comment re system params. I think either this approach or using a Django setting is fine. Both will need a template tag - https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/

Please call the variables ORGANISATION_NAME and ORGANISATION_INITIALISM

File: trade_remedies_caseworker/templates/TRID_abbreviation.html [hunk 1]
```diff
@@ -1 +1 @@
-TRID
\ No newline at end of file
+TRA
```
Comment (davecharles) [inline trade_remedies_caseworker/templates/TRID_abbreviation.html:1]: This file is not used anymore

File: trade_remedies_caseworker/templates/TRID_full_name.html [hunk 2]
```diff
@@ -1 +1 @@
-Trade Remedies Investigations Directorate
\ No newline at end of file
+Trade Remedies Authority
```
Comment (davecharles) [inline trade_remedies_caseworker/templates/TRID_full_name.html:1]: This file is not used anymore

**Review by davecharles — CHANGES_REQUESTED** [review]
I think we can remove the old `TRID_full_name.html` and `TRID_abbreviation.html` files

**Comment by jwgulley** [comment]
> 
> 
> Could we make the name of the product and its initialism a settings variable tied to an env var? That way we can change it any time.
These are now set in local.env, at least for dev purposes - was that what you meant?

**Review by rossmiller — APPROVED** [review]
LGTM!

**Review by davecharles — APPROVED** [review]
LGTM

**Review by rossmiller — CHANGES_REQUESTED** [review]
Could you add the new env vars to local.env.example and an entry in the readme to explain them please?

File: trade_remedies_caseworker/core/templatetags/organisation_initialism.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from core.templatetags import register
+from django.conf import settings
+
+"""
+Template tag to display organisation name and organisation initialism
```
Comment (rossmiller) [inline trade_remedies_caseworker/core/templatetags/organisation_initialism.py:5]: Please change text to "Template tag to display organisation initialism"

**Review by rossmiller — CHANGES_REQUESTED** [review]

File: trade_remedies_caseworker/core/templatetags/organisation_initialism.py (hunk 3 above)
Comment (jwgulley) [reply trade_remedies_caseworker/core/templatetags/organisation_initialism.py]: Thank-you - updated.

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**